packages/grafana-ui/src/themes/GlobalStyles/fonts.ts

```diff
@@ -4,7 +4,9 @@ import { GrafanaTheme2 } from '@grafana/data';
 
 export function getFontStyles(theme: GrafanaTheme2) {
   const grafanaPublicPath = typeof window !== 'undefined' && window.__grafana_public_path__;
-  const fontRoot = grafanaPublicPath ? `${grafanaPublicPath}fonts/` : 'public/fonts/';
+  const relativefontRoot = grafanaPublicPath ? `${grafanaPublicPath}fonts/` : 'public/fonts/';
```

Comment:
[Mentioned in AzDo](https://dev.azure.com/ni/DevCentral/_workitems/edit/3207943#10209516) but looks like emotion's speedy mode does a bunch of weird magic. Curious if [disabling speedy](https://changeset-release--emotion.netlify.app/docs/ssr#puppeteer) (not doing it conditionally as shown but just importing and setting `sheet.speedy(false)`) has an impact on the behavior.

Reply:
Yes, that worked! And as I mentioned in the linked AzDO bug, this is more robust than patching individual relative paths (especially going forward). I didn't notice any impact on rendering speed either.

Reply:
Since we're now disabling this unconditionally, I don't think there's anything we need to add to the README.